### DISCUSSION TITLE
Rotate model 360deg

### DISCUSSION THREAD

**Discussion by cnpog** [google/model-viewer#3437] [Q&A] [answered]
Is it possible to rotate a model up and down like it is possible to turn it left and right? Meaning if I drag the model to the left it spins until my mouse hits the monitor limit, but if I do the same to the top or bottom it stops when I reach the models top/bottom. Can this be changed somehow?

**Comment by elalish** [comment, marked as answer]
See https://github.com/google/model-viewer/discussions/3374#discussioncomment-2587386
